util/updateXML/update1.7.0.0.py

```diff
@@ -42,3 +42,17 @@ def update(jobs):
         )
         for unit_dynamic_model in unit_dynamic_models:
             unit_dynamic_model.set_name("Dynawo.Electrical.Controls.Machines.VoltageRegulators.Standard.SCRX1")
+
+    modelica_models = jobs.dyds.get_modelica_models(lambda modelica_model: modelica_model.get_id() == "Dynawo.Electrical.Controls.Machines.VoltageRegulators.Standard.IEEET1")
+    for modelica_model in modelica_models:
+        unit_dynamic_models1 = modelica_model.get_unit_dynamic_models(lambda unit_dynamic_model: unit_dynamic_model.get_id() == "voltageRegulator")
+        for unit_dynamic_model1 in unit_dynamic_models1:
+            unit_dynamic_model1.init_connects.change_var_name("UStator0Pu", "Us0Pu")
+
+    modelica_models = jobs.dyds.get_modelica_models(lambda _: True)
```

Comment:
Could you add also the one of model_template? Thanks

Reply:
I have just pushed a new version with an additional model_template.